packages/strapi-plugin-content-manager/admin/src/containers/EditPage/saga.js

```diff
@@ -48,6 +45,27 @@ function* dataGet(action) {
   }
 }
 
+function* deleteData() {
```

Comment:
generators? I had seen in other parts `async` & `await`. What is the preference?

Reply:
We use redux-sagas therefore we go for generators instead of async/await.
Here's an [explanation](https://thecodebarbarian.com/redux-saga-vs-async-await.html)